daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -80,6 +80,14 @@ The default value of `spec.metrics.http.increasedCardinality` is `true` in Dapr
 
 Setting `spec.metrics.http.increasedCardinality` to `false` is **recommended** to all Dapr users, to reduce resource consumption. The pre-1.13 behavior, which is used when the option is `true`, is considered legacy and is only maintained for users who have special requirements around backwards-compatibility.
 
+## HTTP Metrics Path Normalization 
+
+Path normalization allows you to manage and control the cardinality of HTTP metrics in Dapr.
+
+This new configuration is opt-in and can be enabled with a new option for the Dapr Configuration resource `spec.metrics.http.pathNormalization.enabled`. When set to `true`, it enables path normalization, which standardizes specified paths for both ingress and egress paths. This reduces the number of unique metrics paths, making metrics more manageable and reducing resource consumption in a controlled way.
```

Comment:
```suggestion
This configuration is opt-in and is enabled via the Dapr configuration `spec.metrics.http.pathNormalization.enabled` setting. When set to `true`, it enables path normalization, which standardizes specified paths for both ingress and egress paths. This reduces the number of unique metrics paths, making metrics more manageable and reducing resource consumption in a controlled way.
```

Reply:
It would be good to have some clear guidance here on what is best to do. I presume the best guidance is to add path normalization to true in order to get metrics showing correctly, is that right

Reply:
added some examples on configuration and the impact on the metrics!